runtime/gc_glue_java/ScavengerRootClearer.cpp

```diff
@@ -265,6 +265,12 @@ MM_ScavengerRootClearer::scavengeContinuationObjects(MM_EnvironmentStandard *env
 #endif /* JAVA_SPEC_VERSION >= 19 */
 }
 
+void
+MM_ScavengerRootClearer::iterateContinuationObjects(MM_EnvironmentBase *env)
+{
+	env->getGCEnvironment()->_continuationObjectBuffer->iterateContinuationObjects(env);
```

Comment:
Still thinking if Buffer is the right class to add it to, but if we do it, it should be a static method, otherwise we may end up calling it with _continuationObjectBuffer NULL. Probably it would still work still the iteration would never access 'this' pointer, but still it would be cleaner. 

Being a static, it could not be a virtual, but that would still be ok, (Scavenger/Marker would explicitly call Standard variant of iteration, and CopyForwad/GlobalMarkingScheme would  call VLHGC variant.)

RootScanner's iterate method state in base class as empty/pure virtual, so we can share the jit's condition.